test/util/test_node.py

```diff
@@ -21,13 +22,39 @@ async def test_configuration_parameter_values(
     config_values = node.get_configuration_values()
     assert len(config_values) == 12
 
+    assert node_2.node_id == 31
+    config_values_2 = node_2.get_configuration_values()
+    assert len(config_values_2) == 15
+
     for value in config_values.values():
         assert isinstance(value, ConfigurationValue)
 
     # Test setting a configuration parameter that has no metadata
     with pytest.raises(NotImplementedError):
         await async_set_config_parameter(node, 1, 2)
 
+    # Test setting an manual entry configuration parameter with an invalid value
+    with pytest.raises(InvalidNewValue):
+        await async_set_config_parameter(node_2, "Purple", 8, 255)
+
+    # Test setting an manual entry configuration parameter with an valid value
+    ack_commands_2 = mock_command(
+        {"command": "node.set_value", "nodeId": node_2.node_id},
+        {"success": True},
+    )
+
+    await async_set_config_parameter(node_2, 190, 8, 255)
+
+    value = node_2.values["31-112-0-8-255"]
+    assert len(ack_commands_2) == 1
+    assert ack_commands_2[0] == {
+        "command": "node.set_value",
+        "nodeId": node_2.node_id,
+        "valueId": value.data,
+        "value": 190,
+        "messageId": uuid4,
+    }
+
```

Comment:
Can we add an additional success test for setting this value using a valid state label?

Reply:
Sure